final_prep/README.md

```diff
@@ -26,30 +26,30 @@ Take your time as you go back and refactor your exercises from each section. We'
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/))
 - [ ] Study/Project work time
 - [ ] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/58535045/118335365-548c4280-b4c4-11eb-89fa-503334b5d877.png "Week 1")
```

Comment:
Nice work here- I like the color coding, and looks like you have appropriate expectations for the workload. Now that Turing is approaching, you can link up to the Mod1 calendar, so you have ideas of the material for the week.